app/controllers/space_contacts_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class SpaceContactsController < AuthenticateController
+  before_action :set_space_contact, except: [:create]
+
+  def create
+    @space_contact = SpaceContact.create!(space_contact_params)
+
+    if @space_contact.save
+      redirect_to space_path(@space_contact.space_id), notice: t('space_contacts.contact_created')
+    else
+      render turbo_stream: turbo_stream.replace(
+        @space_contact,
+        partial: 'space_contacts/form',
+        locals: { space_contact: @space_contact }
+      )
+    end
+  end
+
+  def update
+    if @space_contact.update(space_contact_params)
+      redirect_to space_path(@space_contact.space_id), notice: t('space_contacts.contact_updated')
```

Comment:
The `notice` method for giving user feedback on actions is something I haven't used in my controllers, and should have! Will try to implement it as I go forward! :) 

One question I sit with seeing it now is: Did you notice (hehe) that I didn't use  `notice` when reviewing my PRs? Missing out on methods that are common to the code base is stuff I would love to get picked up on in reviews!

Reply:
It is a different way to use notices than this way. I'll tag you on it, have to make them "turbo" first.